raco/algebra.py

```diff
@@ -1092,8 +1171,13 @@ def num_tuples(self):
     def shortStr(self):
         return self.opname()
 
+    def partitioning(self):
```

Comment:
Just a random thought: if hash functions were multivalued then maybe they could just emit all workers to simulate broadcast (and a subset of workers for replication and a constant worker for collect)?